web/rootfs/defaults/meet.conf

```diff
@@ -22,6 +22,10 @@ add_header X-XSS-Protection "1; mode=block";
 add_header X-Jitsi-Shard {{ .Env.DEPLOYMENTINFO_SHARD }};
 {{ end }}
 
+{{ if .Env.DISABLE_FLOC | default "1" | toBool }}
```

Comment:
This is very counter intuitive. Since it's opt-in, let's call it ENABLE_FLOC instead, and default to 0.

Reply:
Thank you for the advice, inverted it in 9c5a4ac (incl. wording and commit msg)